### PR TITLE
Merge sysbox-ee docs into sysbox docs

### PR DESCRIPTION
**Pull request by ctalledo**

### CODE REVIEW

File: docs/quickstart/kind.md [hunk 1]
```diff
@@ -18,10 +18,49 @@ detail.
...
+Other approaches at creating K8s-in-Docker clusters (e.g., K8s.io KinD) are good
+but use very unsecure privileged containers and require customized images. With
+Sysbox, you get well isolated containers and there are no special image
+requirements (i.e., you fully control the container image).
+
+This is an excellent choice for CI/CD, local testing, and learning environments.
+It's a quick, very efficient, cost effective, and secure way of creating K8s
+clusters, compared to using cloud-hosted K8s clusters or local virtual machines.
+
+As Sysbox continues to mature, we expect K8s-in-Docker will be used in in
```
Comment (rodnymolina) [inline docs/quickstart/kind.md:38]: double "in" preposition.

File: docs/user-guide/security.md [hunk 2]
```diff
@@ -116,27 +116,28 @@ There is one advantage of Directed userns ID mapping:
...
 
-But there is a drawback:
+But there are a couple of drawbacks:
 
 -   Configuring Docker with userns-remap places a few [functional limitations](https://docs.docker.com/engine/security/userns-remap/#user-namespace-known-limitations)
     on regular Docker containers (those launched with Docker's default runc).
 
+-   Docker assigns the same user-ID mapping to all containers. This is not ideal:
+    it would be better to assign each container exclusive user-ID mappings for
+    improved cross-container isolation.
```
Comment (rodnymolina) [inline docs/user-guide/security.md:126]: This point is only relevant in sysbox-EE and not in sysbox-CE so this sentence may need clarification.

**Review by rodnymolina — APPROVED** [review]
LG. Have only minor comments.

File: docs/quickstart/kind.md (hunk 1 above)
Comment (ctalledo) [reply docs/quickstart/kind.md:38]: Fixed, thanks.

File: docs/user-guide/security.md (hunk 2 above)
Comment (ctalledo) [reply docs/user-guide/security.md]: Fixed, thanks!

**Pull request merged**